src/blocklypropclient.js

```diff
@@ -373,16 +332,20 @@ function establish_socket() {
             if (ws_msg.type === 'hello-client') {
                 // type: 'hello-client',
                 // version: [String version (semantic versioning)]
-                checkClientVersionModal(ws_msg.version);
+                // rxBase64: [boolean, accepts base64-encoded serial comms (all versions transmit base64)]
```

Comment:
There is a bug in the launcher (and this is a related workaround).  "No device" messages are not being encoded by the Launcher before they are transmitted.  Also, the Launcher was not set up to decode transmissions from the client, so this variable is to allow the launcher to tell the browser if sent messages should be encoded, which avoids version checking for this feature.